src/streamlink/plugins/canalplus.py

```diff
@@ -6,23 +6,36 @@
 
 
 class CanalPlus(Plugin):
-    API_URL = 'http://service.canal-plus.com/video/rest/getVideos/{0}/{1}?format=json'
-    CHANNEL_MAP = {'canalplus': 'cplus', 'c8': 'd8', 'cstar': 'd17', 'cnews': 'itele'}
```

Comment:
i don't think you can remove everything off this,

http://service.canal-plus.com/video/rest/getVideosLiees/itele/203105?format=json
http://www.cnews.fr/direct

still needs this.